eng/ignore-links.txt

```diff
@@ -28,3 +28,5 @@ https://docs.microsoft.com/javascript/api/@azure/arm-peering
 https://docs.microsoft.com/javascript/api/@azure/arm-peering?view=azure-node-preview
 https://docs.microsoft.com/javascript/api/@azure/arm-postgresql-flexible
 https://docs.microsoft.com/javascript/api/@azure/arm-postgresql-flexible?view=azure-node-preview
+https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/deviceupdate/iot-device-update-rest/samples/v1/javascript/devices.js
```

Comment:
@praveenkuttappan any idea why these need to be ignored?

Reply:
I find the root cause, the url is wrong, and I have fixed them, then we don't need to add them to ignore.